mlir/test/mlir-miopen-driver/tuning_xdlops.mlir

```diff
@@ -2,21 +2,13 @@
 
 // RUN: miopen-gen -p -x2 | mlir-miopen-driver -miopen-affix-params -miopen-conv-to-gemm | FileCheck %s --check-prefix=STEP1
 // RUN: miopen-gen -p -x2 | mlir-miopen-driver -miopen-affix-params -miopen-conv-to-gemm -miopen-gridwise-gemm-to-blockwise | FileCheck %s --check-prefix=STEP2
-// RUN: miopen-gen -p --fil_layout=kyxc --in_layout=nhwc --out_layout=nhwk -x2 | mlir-miopen-driver -miopen-affix-params -miopen-conv-to-gemm | FileCheck %s --check-prefix=NHWC
 
 // STEP1: mPerWave
 // STEP1: nPerWave
-// STEP1: matrix_c_data_per_copy = 4
-// STEP1: matrix_c_dest_vector_write_dim = 4
-// STEP1: matrix_c_source_vector_read_dim = 2
```

Comment:
So if we want to keep the spirit of this test in, I'd go to `--gridwise-gemm-to-blockwise` and look for a `threadwise_copy_v2` with length `4`